File: Clase5/Python/tarea_bucles.py
##1. Hacer un programa que imprima la tabla de multiplicar del 1 al 10 usando for.

numero = int(input('que numero quieres multiplicar del 1 al 10: '))

for i in range(1,11):
    print(f'{i} * {numero}= {i * numero}')


## 2. Crear una calculadora simple con while que repita operaciones (suma,resta,multiplicación) y division hasta que el usuario decida salir.    
numero1 = float(input("Introduce un numero: "))
numero2 = float(input("Introduce un numero: "))

operacion= input("Que operación quieres realizar (suma,resta,multiplicación o división):")

resultado = 0

if operacion == 'suma':
    resultado = numero1 + numero2
elif operacion == 'resta':
    resultado = numero1 - numero2
elif operacion == 'multiplicacion':
    resultado = numero1 * numero2
elif operacion == 'division':
    resultado = numero1 / numero2
print("El resultado es", resultado)

##Tarea con while
while True:
  print("Que operación quieres hacer")
  print("1. Suma")
  print("2. Resta")
  print("3. Multiplicación")
  print("4. División")
  print("5. Salir")

  operacion = input("Ingrese una opción con numero: ")

  if operacion == '1':
    print("Eligio suma")
    numero1 = int(input("Ingrese el primer numero: "))
    numero2 = int(input("Ingrese el segundo numero: "))
    print(numero1 + numero2)
    break
  elif operacion == '2':
    print("Eligio resta")
    numero1 = int(input("Ingrese el primer numero: "))
    numero2 = int(input("Ingrese el segundo numero: "))
    print(numero1 - numero2)
    break
  elif operacion == '3':
    print("Eligio multiplicacion")
    numero1 = int(input("Ingrese el primer numero: "))
    numero2 = int(input("Ingrese el segundo numero: "))
    print(numero1 * numero2)
    break
  elif operacion == '4':
    print("Eligio division")
    numero1 = int(input("Ingrese el primer numero: "))
    numero2 =int(input("Ingrese el segundo numero: "))
    if numero2 == 0:
      numero2 = int(input("No se puede dividir por 0 ingrese otro numero"))
    elif numero2 > 0:
      print(numero1/numero2)
      break
  elif operacion == '5':
    print("Saliendo del programa")
    break
  elif operacion != '1'or operacion != '2'or operacion != '3'or operacion != '4'or operacion != '5':
    print("Debe elegir un numero entre 1 y 5")

File: Clase5/Java/tarea_bucles.java
import java.util.Scanner;

public class tarea_bucles {
    public static void main(String[] args) {

        Scanner sc=new Scanner(System.in);
        int operacion;
        double numero1;
        double numero2;

        do {
            System.out.println("seleccione una operación: ");
            System.out.println("1. Suma");
            System.out.println("2. resta");
            System.out.println("3. multiplicacion");
            System.out.println("4. division");
            System.out.println("5. Salir");
            System.out.print("seleccione una opcion:");
            operacion = sc.nextInt();

            if (operacion >=1 && operacion <= 4) {
                System.out.print("Ingrese el primer numero:");
                numero1 = sc.nextDouble();
                System.out.print("Ingrese el segundo numero:");
                numero2 = sc.nextDouble();

                switch (operacion) {
                    case 1:
                        System.out.println("Resultado de la suma:" + (numero1 + numero2));
                        break;
                    case 2:
                        System.out.println("Resultado de la resta:" + (numero1 - numero2));
                        break;
                    case 3:
                        System.out.println("Resultado de la multiplicacion:" + (numero1 * numero2));
                        break;
                    case 4:
                        if (numero2 !=0){
                        System.out.println("Resultado de la division:" + (numero1 / numero2));
                        } else {
                            System.out.println("Error: No se puede dividir por cero");
                        }
                        break;
                
                    default:
                        break;
                }          

            } else if (operacion == 5){
                System.out.println("Gracias por usar la calculadora");
            }

        } while (operacion != 5);
        sc.close();
        

    }
}



